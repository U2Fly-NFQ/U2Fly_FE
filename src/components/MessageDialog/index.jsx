import { message } from 'antd'

const success = (msg) => {
  message.success({
    content: { msg },
    className: 'custom-class',
    style: {
      marginTop: '20vh',
    },
  })
}

export default success
