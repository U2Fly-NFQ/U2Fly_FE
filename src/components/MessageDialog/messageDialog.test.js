import renderer from 'react-test-renderer'
import React from 'react'
import MessageDialog from './'

describe('Message dialog page test', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<MessageDialog msg={'Hello'} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
