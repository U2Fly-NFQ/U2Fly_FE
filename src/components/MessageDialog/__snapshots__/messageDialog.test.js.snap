// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page test renders correctly 1`] = `null`;

exports[`Message dialog page test renders correctly 1`] = `null`;
